dragonfly/implementations/psyllid_provider.py

```diff
@@ -233,51 +238,167 @@ def get_number_of_streams(self, channel):
         return stream_count
 
 
-    # These methods shoud not be used yet
-    def set_fmt_snr_threshold(self, snr, channel='a'):
-        if self.mode_dict[channel] == 'streaming':
-            logger.warning('Psyllid not in streaming mode')
-            return False
-        request = '.node-config.ch'+str(self.channel_dict[channel])+'.fmt.threshold-power-snr'
-        result = self.provider.set(self.queue_dict[channel]+request, snr)
-        self.deactivate(channel)
-        time.sleep(1)
-        self.activate(channel)
-        return True
+    # trigger control
+    def set_trigger_configuration(self, channel='a', threshold=16, threshold_high=0, n_triggers=1,):
```

Comment:
They are not used by someone. They are used by the roach_daq_run_interface. If you *set_trigger_configuration* there it will first call *psyllid_interface.set_trigger_configuration* and then *psyllid_interface.get_trigger_configuration* and return the result. The operator will see what settings were applied.

Reply:
I disagree with the statement that there should be a return. A set function should not return anything. It should assign the value. If there is an error it should raise an exception, no exception implies success.

Reply:
If we ever try to do set_and_check (https://github.com/project8/dragonfly/issues/47), everything will be terrible.

_This is way out of scope for this PR._

Reply:
As defined here, this will be used via "dragonfly cmd" and I thought we want cmd to return stuff.
If this was something used via 'dragonfly set' I would agree, but this is not (or at least not as currently implemented).